src/UseCase/BannerProvideUseCase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\UseCase;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @license GNU GPL v2+
+ */
+class BannerProvideUseCase {
```

Comment:
IMO the use case should read more like a sentence, with VerbObject, e.g. `ProvideBannerUseCase`. And do we really "provide" it or do we "select" it? I'd think the latter. Since we're now beoming the domain experts and need to talk the same language this beomes important, even if it means that we talk a lot about naming.